test/configCases/externals/optional-externals-array/webpack.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+	externals: {
+		external: ["./math", "subtract"]
```

Comment:
You can use `["webpack", "version"]` and check if the result is a string.

Checking the error doesn't really check if `subtract` is used or ignored.

Reply:
I just copied what is in the other `test/configCases/externals/optional-externals-*/webpack.config.js` tests. These tests are just testing the configuration, I would expect there to be other tests that actually verify the functionality.

Reply:
Just tried `["webpack", "version"]` and it still errors (test passes without alteration). So if that is supposed to work, there's another issue (which we should then follow up on in #8041 I guess).

Reply:
Did you follow the CONTRIBUTION instructions about `yarn link && yarn link webpack`?

Reply:
Followed the instructions, same result. Looks like CI failed due to a random timeout (only AppVeyor failed). Will push commit again.